nym-vpn-app/src/screens/home/useStreamingOptimizedLabel.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router';
+import { useTranslation } from 'react-i18next';
+import { ButtonText } from '../../ui/index';
+import { routes } from '../../router';
+import { useInAppNotify, useMainDispatch, useMainState } from '../../contexts';
+import { StateDispatch } from '../../types/index';
+import { setStreamOptimizedLabelSeen } from './util';
+
+const StreamingOptimizedLabelContent = () => {
+  const { t } = useTranslation('notifications');
+  const navigate = useNavigate();
+  const dispatch = useMainDispatch() as StateDispatch;
+
+  const handleClick = () => {
+    setStreamOptimizedLabelSeen(dispatch);
+    navigate(routes.exitNodeLocation);
+  };
+
+  return (
+    <div className="flex flex-row justify-between gap-5 items-center">
+      <span className="dark:text-white text-baltic-sea">
+        {t('streaming-optimized-label')}
+      </span>
+      <ButtonText
+        color="transparent"
+        className="text-malachite"
```

Comment:
Instead of passing arbitrary className to override text color, you should extend the `ButtonText` prop `color` API, add a new one eg. "malachite", see `nym-vpn-app/src/ui/ButtonText.tsx`